cmd/chainbridge/main.go

```diff
@@ -227,6 +227,12 @@ func run(ctx *cli.Context) error {
 		h := health.NewHealthServer(port, c.Registry, int(blockTimeout))
 
 		go func() {
+			http.HandleFunc("/liveness", func(w http.ResponseWriter, r *http.Request) {
+				fmt.Fprintf(w, "Liveness probe is ok")
+			})
+			http.HandleFunc("/readiness", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Update endpoints according to the tech design

Reply:
fixed